src/libs/actions/MemoryOnlyKeys/MemoryOnlyKeys.js

```diff
@@ -0,0 +1,18 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../../ONYXKEYS';
+
+const memoryOnlyKeys = [ONYXKEYS.COLLECTION.REPORT, ONYXKEYS.COLLECTION.POLICY, ONYXKEYS.PERSONAL_DETAILS_LIST];
+
+const enable = () => {
+    console.debug('[MemoryOnlyKeys] enabled');
```

Comment:
Maybe this (and the one belwo) could be a `Log.info()`? Could help show that someone enabled this mode if we are investigating server logs.

Reply:
That's a good idea